lib/assessment.md

```diff
@@ -41,7 +50,7 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+The example we got in class used div as a selector. That seems like it might work with any of these. Not sure what else could be a selector?
```

Comment:
A selector is just a way to refer to an element of group of elements. Take some time and review that part of the HTML and CSS lesson.